finetuner/client/base.py

```diff
@@ -1,23 +1,39 @@
+import json
 import os
 from typing import Optional
 
+import hubble
 import requests
 from path import Path
 
-from finetuner.constants import HOST
+from finetuner.constants import (
+    AUTHORIZATION,
+    CHARSET,
+    DATA,
+    HOST,
+    HUBBLE_USER_ID,
+    TOKEN_PREFIX,
+    UTF_8,
+)
 
 
 class BaseClient(object):
-    """Base Finetuner API client.
+    """Base Finetuner API client."""
 
-    :param user_id: Unique identifier of a user.
-    """
-
-    def __init__(self, user_id):
-        self._user_id = user_id
-        self._session = requests.session()
+    def __init__(self):
         self._base_url = Path(os.environ.get(HOST))
-        self._session.headers.update({'Accept-Charset': 'utf-8'})
+        self._session = self._get_client_session()
+        self._hubble_client = hubble.Client(max_retries=None, timeout=10, jsonify=True)
+        self._hubble_user_id = json.loads(self._hubble_client.get_user_info())[DATA][
+            HUBBLE_USER_ID
+        ]
```

Comment:
what will happen if we get a 404 here? json load `[DATA][HUBBLE_USER_ID]` will break, right?